src/main/java/org/springframework/samples/petclinic/rest/importcsv/ImportCSV.java

```diff
@@ -26,114 +28,239 @@
     @Autowired
     private ClinicService clinicService;
 
+    private static final int COMMAND_FIELD_NUMBER = 5;
+    private static final String COMMAND_FORMAT = "name;date(yyyy-MM-dd);type id;owner's id;command";
+    private static final String COMMAND_DELIMITER = ";";
+
+    private static final int PET_NAME_INDEX = 0;
+    private static final int DATE_INDEX = 1;
+    private static final int TYPE_INDEX = 2;
+    private static final int OWNER_ID_INDEX = 3;
+    private static final int COMMAND_TYPE_INDEX = 4;
+
+    private static final String PET_NAME_REGEX = "^[a-zA-Z]+$";
+    private static Pattern namePattern = Pattern.compile(PET_NAME_REGEX);
+
+    private static final String DATE_PATTERN = "yyyy-MM-dd";
+
+    private static Set<String> supportedCommands = new HashSet<>();
```

Comment:
nice but where is javadoc?

Reply:
I like that you created variables to ease configuration - since the parameters are private, javadoc is not mandatory in my opinion